src/Layout/Presentation/ElectionOverview/ElectionOverview.tsx

```diff
@@ -125,68 +126,72 @@ export class ElectionOverview extends React.Component<ElectionOverviewProps, {}>
                     {...norwegian}
                     columns={[
                         {
-                            Header: "Parti",
+                            Header: <span className="is-pulled-left">Parti</span> ,
                             accessor: "partyCode",
                             filterMethod: caseInsensitiveFilterMethod,
-                            Footer: <strong>Utvalg</strong>,
+                            Footer: <strong className="is-pulled-left">Utvalg</strong>,
                             Cell: (row) => {
-                                return row.original.partyName ? (
-                                    <abbr title={row.original.partyName}>{row.value}</abbr>
-                                ) : (
-                                    row.value
-                                );
+                                return <span className="is-pulled-left">{row.original.partyName}</span>;
                             },
                         },
                         {
-                            Header: "Stemmer",
+                            Header: <span className="is-pulled-right">Stemmer</span> ,
                             accessor: "votes",
                             filterable: false,
-                            Footer: <strong>{data.map((value) => value.votes).reduce(toSum, 0)}</strong>,
+                            Cell: (row) => {
+                                return numberFormat(row.value);
+                            },
+                            Footer: <strong>{numberFormat(data.map((value) => value.votes).reduce(toSum, 0))}</strong>,
                         },
                         {
-                            Header: "%",
+                            Header: <span className="is-pulled-right">Oppsluting %</span> ,
                             id: "%",
                             Filter: selectFilterWithOptions(thresholdOptions),
                             filterMethod: thresholdFilterMethod(this.props.threshold),
                             accessor: (d: ElectionOverviewDatum) => roundNumber(d.percentVotes, decimals),
+                            Cell: (row) => {
+                                return numberFormatFraction(row.value,decimals);
```

Comment:
I noticed you are first rounding the number on line 151, then again limiting the number of decimals on 153. Is the initial rounding superfluous?
On line 186 you are skipping the initial rounding.

Reply:
Correct, the initial rounding superfluous.